rotkehlchen/tests/unit/globaldb/test_globaldb_cache.py

```diff
@@ -243,23 +250,25 @@ def test_curve_cache(rotkehlchen_instance, use_curve_api):
         token_type=EvmTokenKind.ERC20,
     ))
 
+    pools, gauges = read_curve_pools_and_gauges()
+    for pool in CURVE_EXPECTED_LP_TOKENS_TO_POOLS.values():
+        assert pool not in pools
+    for pool in CURVE_EXPECTED_POOL_COINS:
+        assert pool not in pools
+    assert len(gauges & CURVE_EXPECTED_GAUGES) == 0
+
     # check that it was deleted successfully
     token = GlobalDBHandler().get_evm_token(
         address='0x3Ed3B47Dd13EC9a98b44e6204A523E766B225811',
         chain_id=ChainID.ETHEREUM,
     )
     assert token is None
 
```

Comment:
They are all part of the same check. Put them together. No need for the comment.

```suggestion
    assert GlobalDBHandler().get_evm_token(
        address='0x3Ed3B47Dd13EC9a98b44e6204A523E766B225811',
        chain_id=ChainID.ETHEREUM,
    ) is None
```